Sources/Plasma/FeatureLib/pfDXPipeline/plDXTextFont.cpp

```diff
@@ -63,10 +63,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 //// Local Stuff //////////////////////////////////////////////////////////////
 
-static const long PLD3D_FONTFVF = D3DFVF_XYZ | D3DFVF_DIFFUSE 
+static constexpr long PLD3D_FONTFVF = D3DFVF_XYZ | D3DFVF_DIFFUSE 
                                 | D3DFVF_TEX1 | D3DFVF_TEXCOORDSIZE3(0);
 
-static D3DMATRIX d3dIdentityMatrix{ 1.0f, 0.0f, 0.0f, 0.0f,
+static constexpr D3DMATRIX d3dIdentityMatrix{ 1.0f, 0.0f, 0.0f, 0.0f,
```

Comment:
I don't know if the identity matrix should be constexpr. Something being both static and constexpr may already be a bit of a paradox. But the identity matrix is loaded as a buffer from a pointer, which means it can't actually be constexpr easily by a compiler. And it might be more optimal just to have one copy of the buffer in memory anyway.

Reply:
`static constexpr` is perfectly valid. Sometimes you have things that can be used at both compile-time and runtime. In this case, we aren't doing any compile-time matrix math, so it probably makes more sense to just leave this one alone. I doubt that the generated machine code is any different since we're just going to be pushing a pointer onto the stack.

Reply:
i can revert this if desired